homeassistant/components/modbus/modbus.py

```diff
@@ -93,7 +93,10 @@ def write_coil(service):
         address = service.data[ATTR_ADDRESS]
         state = service.data[ATTR_STATE]
         client_name = service.data[ATTR_HUB]
-        hub_collect[client_name].write_coil(unit, address, state)
+        if isinstance(state, list):
```

Comment:
The state is validated as a boolean in the service schema. How would it be a list?

Reply:
ups...I isolated this PR from a bigger PR (in preparation) that add testing of write calls, but I forgot this one, sorry about that.

Thanks for catching it.